Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/features/settings/ui/SettingsAdapter.java

```diff
@@ -188,6 +201,41 @@ public void onBooleanClick(CheckBoxSetting item, int position, boolean checked)
     mView.onSettingChanged();
   }
 
+  public void onInputStringClick(InputStringSetting item, int position)
+  {
+    final EditText input = new EditText(mContext);
+    input.setText(item.getSelectedValue(getSettings()));
+
+    LinearLayout layout = new LinearLayout(mContext);
+    layout.setOrientation(LinearLayout.VERTICAL);
+    LinearLayout.LayoutParams params = new LinearLayout.LayoutParams(
+      LinearLayout.LayoutParams.MATCH_PARENT, LinearLayout.LayoutParams.WRAP_CONTENT);
+    params.setMargins(50, 0, 50, 0);
```

Comment:
I think it would be better to define all this in an XML file than directly in the code.

Reply:
Tried making this into an xml, I'm not very familiar with Android Dev, or Frontend Dev in general so I hope I did it right